src/screens/market/table/right-swipe-view/right-swipe-view.selectors.ts

```diff
@@ -0,0 +1,4 @@
+export enum RightSwipeViewSelectors {
+  ToggleFavouriteToken = 'Toggle favourite token',
+  BuyToken = 'Buy token'
```

Comment:
we don't have structure for `testID`s yet, but you need to add into the name what component is this (for ex `BuyTokenButton` instead of `BuyToken`) and add what screen is this into value, cuz there could be conflicts between different screens (`Market/Buy token button` instead of `Buy token`)